tensorflow/core/graph/mkl_graph_util.h

```diff
@@ -122,6 +125,23 @@ inline string GetMklEagerOpName(const string& name) {
   return string(kMklEagerOpPrefix) + name;
 }
 
+#ifdef ENABLE_INTEL_MKL_BFLOAT16
+static inline bool CheckBfloat16Support(DataType T) {
```

Comment:
If this is specific to `bfloat16` type, why do we have to take the type as an input parameter? It could just be `CheckBfloat16Support()`.

Nit: Also, I think `HasBfloat16Support()` might make it a bit clearer that the function is returning `true` if there is `bfloat16` support.

Reply:
Done changed to HasBfloat16Support(). Thank you for the suggestion.

As for as the Type extra check, we wanted the message to be print only when running the BFloat models and not when running the FP32 op models even when the compiler flag is true. (if the model still crashes, the problem is something different). Extra qualifications ensure that message is printed only when intended.

Reply:
I see. This is different from `HasBfloat16Support` that I imagine then (should return true if AVX512F regardless of the datatype we are focusing on, because it's about the machine). Maybe we should keep it `CheckBfloat16Support`. Please see my other comment.